db/main/migrate/20180222000000_add_com_id_and_org_id.rb

```diff
@@ -0,0 +1,27 @@
+class AddComIdAndOrgId < ActiveRecord::Migration
+  def change
+    add_column :branches, :org_id, :integer
```

Comment:
You might want to add `NOT_NULL` to these as well.

Reply:
@aakritigupta no, i don't think so. they're going to be `null` quite often?

Reply:
please make sure all new columns are nullable, as adding a `not null` column to a large table in production is not a safe operation. see also: https://www.braintreepayments.com/blog/safe-operations-for-high-volume-postgresql/

Reply:
That was a good read, thanks for pointing to it @igorwwwwwwwwwwwwwwwwwwww 

@svenfuchs the column won't be `null` often and it is not worth adding the default value in a migration, so, let's keep it as is :)